gitdata.py

```diff
@@ -14,6 +108,7 @@ def __init__(self, login):
 
     def __str__(self):
         return f"{self.login}"
+>>>>>>> main
```

Comment:
These text markers get added to the .py files when you do merge operations. You must delete these lines from the code or else it won't run.

Reply:
I see, I will remove those in the future